src/VirtualClient/VirtualClient.Actions/SPECcpu/SpecCpuExecutor.cs

```diff
@@ -443,9 +443,15 @@ private async Task<string> GetInstalledCompilerDumpVersionAsync(string compilerN
         {
             string command = compilerName;
             string commandArguments = "-dumpversion";
-
             string version = string.Empty;
 
+            if (this.Platform == PlatformID.Win32NT)
+            {
```

Comment:
On windows please add fault tolerance, and if bvoth ChocolateyToolsLocation wasn't found and gcc also wasn't found, assume compiler version is 10 (use gcc>10 workaround)